### ISSUE TITLE
Email newsletter signup section

### ISSUE THREAD

**Issue by brashrebel** [realbig/RBM_TH_PATROBERTSON#24]
One very important feature on the site which I forgot about is the newsletter signup. Something functionally like what we have in the footer of RBP. There should be a section on the home page and something persistent in either the sidebar or footer.

**Comment by d4mation** [comment]
What newsletter service does Pat use/will she use? That will shape largely how this feature pans out.

**Comment by joelworsham** [comment]
Also note that this _is_ in the mock. 
<img width="1158" alt="screen shot 2016-10-28 at 3 33 14 pm" src="https://cloud.githubusercontent.com/assets/4182573/19819715/e117fb62-9d23-11e6-835b-b2cc9e1ff7cc.png">

**Comment by d4mation** [comment]
@joelworsham Yeah, I think I may have pushed it aside as I didn't know which newsletter service to use and then it slipped my mind.

**Comment by d4mation** [comment]
I've nearly got this set, but I've encountered a slight problem. [I'm using MailChimp's official WP Plugin to create the Subscription Form.](https://wordpress.org/plugins/mailchimp/)

The Submit button is `input[type="submit"]`, which you [cannot apply `:after` Pseudo-Selectors to](http://stackoverflow.com/a/2591460). That rules out adding the Font Awesome Icon via CSS.

If I set the submit text to include HTML directly, WP decides to sanitize it into text rather than HTML.

The way the Widget is created is... pretty poorly. It pulls values from `wp_options` via a Settings Page rather than actual Widget Options. The Widget itself doesn't have any Hooks I can utilize either.

Aside from adding it in there using JavaScript after page load (ew), all I can think of doing would be to extend the Widget Class and create a slightly different version of the Widget to include additional fields so that the icon could be included properly. However, that could lead to incompatibility issues as the parent Widget receives updates.

![image](https://cloud.githubusercontent.com/assets/7770631/19861877/942de562-9f64-11e6-9b27-81dc2510d3bb.png)
